README.md

```diff
@@ -44,12 +44,15 @@ Read the [introduction](introduction.md).
 ## Installation
 
 This module is distributed via [npm][npm] which is bundled with [node][node] and
-should be installed as one of your project's `devDependencies`:
+should be installed as one of your project's `dependencies`:
 
 ```
-npm install react-ideal-image react-waypoint --save
+npm install react-ideal-image --save
 ```
 
+> This package also depends on `react`, `prop-types`, and `react-waypoint`.
```

Comment:
Why not simply `npm install react-ideal-image react-waypoint prop-types --save`?

Reply:
Either works I guess 😄 I am seeing that you draw a lot of inspiration from Kent C. Dodds and his Downshift project [does it that way](https://github.com/paypal/downshift/blob/master/README.md#installation).

Happy to change it to what you suggest though.

Reply:
I guess both ways ok. I just thought it is easier to copy/paste `npm install ...` to get started